manifests/l/LANCommander/Launcher/1.1.5/LANCommander.Launcher.locale.en-US.yaml

```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: LANCommander.Launcher
+PackageVersion: 1.1.5
+PackageLocale: en-US
+Publisher: LANCommander
+PublisherUrl: https://github.com/LANCommander
+PublisherSupportUrl: https://github.com/LANCommander/LANCommander/issues
+PackageName: LANCommander Launcher
+PackageUrl: https://github.com/LANCommander/LANCommander
+License: MIT
+ShortDescription: LANCommander Launcher Setup

```

Comment:
Please provide a better description of what LANCommander Launcher is, what it does, or what it is used for